test/address_sanitizer_setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
some os do not have bash in `/bin` 
```suggestion
#!/usr/bin/env bash
```

Reply:
Fixed in commit below.